src/opm/input/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -377,7 +378,14 @@ namespace Opm {
                 double multFlt = faultRecord.getItem(1).get< double >(0);
                 try
                 {
-                    m_faults.setTransMult( faultName , multFlt );
+                    if (edit && m_faults.hasFault(faultName) &&
+                                inserted.find(faultName) == inserted.end()) {
+                        multFlt *= m_faults.getFault(faultName).getTransMult();
+                    }
+                    if (edit) {
+                        inserted.insert(faultName);
+                    }
+                    m_faults.setTransMult(faultName, multFlt);
```

Comment:
Assuming that the default multiplier is one, I think this will work if there is only one occurrence for a fault in the EDIT section. For multiple occurrences it  seems like this will use the first occurrence in the EDIT section  for the fault and ignore the later ones, but actually the last one should be used.

Reply:
There is no default multiplier. When there are no GRID entries, I need the inserted set to track that we insert from EDIT and not multiply, precisely so we get the last entry. But there is still some uncertainty about whether or not to apply multiple multiplicative EDIT entries or not. Currently it takes the first and only the first, but that might have to be revised.